Core/ControlMapper.cpp

```diff
@@ -386,6 +423,36 @@ bool ControlMapper::Key(const KeyInput &key, bool *pauseTrigger) {
 	return UpdatePSPState(mapping);
 }
 
+void ControlMapper::ToggleSwapAxes() {
+	swapAxes_ = !swapAxes_;
+
+	// To avoid stuck keys, just reset it all.
+	for (auto &mapping : curInput_) {
+		mapping.second = 0.0f;
+	}
```

Comment:
Hm, suppose I had a mapping for L1+R1 (together) and also had R2 mapped to swap.  I start by pressing (and holding) L1, then I press R2 (swapping) and then press R1.  Wouldn't this clear L1 and thus make my L1+R1 press fail?

I guess I'm not really sure what one would expect, though.  If I had L1 in the above example mapped to Up, you'd think it should then release Up an set Analog Y to -1.0, I suppose... not sure what it did before.  I haven't used this feature.

-[Unknown]